src/main/java/org/broadinstitute/hellbender/tools/exome/TargetArgumentCollection.java

```diff
@@ -31,7 +32,7 @@
  * </p>
  * <p>
  *   If even this mechanism does not provide a target file name, and the target file is deemed non-optional,
- *   the a {@link UserException.BadArgumentValue} is thrown.
+ *   the a {@link CommandLineException.BadArgumentValue} is thrown.
```

Comment:
Yes, depending on what packages you pass to the javadoc run. When I did it today I just ran the javadoc task in gatk-protected, which I assumed is only passing the packages in gatk-protected, since thats all that came out in the output. So if you do that, UserException would not be known to the javadoc task, and it wouldn't generate a link. In theory we could do a single combined run on everything, but I think thats not how its currently set up.